code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -591,7 +591,37 @@
 	gear_tweaks += new /datum/gear_tweak/path(temperature)
 
 /datum/gear/accessory/necklace
-	display_name = "necklace (colorable)"
-	description = "A piece of jewelry that goes around your neck."
+	display_name = "colored necklace selection"
```

Comment:
```suggestion
	display_name = "necklace selection (colorable)"
```

Reply:
colored here implies they are already colored in the icon and thus not colorable as an option. If there's a better way to word this (im sure there is) let me know

Reply:
misunderstood the naming. it's fine as is.